src/components/organisms/AssetActions/Edit/index.tsx

```diff
@@ -124,7 +143,8 @@ export default function Edit({
     <Formik
       initialValues={getInitialValues(
         metadata,
-        ddo.findServiceByType('access').attributes.main.timeout
+        ddo.findServiceByType('access').attributes.main.timeout,
+        ddo.price.value
```

Comment:
we are within the `AssetProvider` so we should be able to not rely on the DDO price value for future reasons. So rather use the `price.value` from `useAsset()` instead